### ISSUE TITLE
What about a langchain wrapper for lit-llama?

### ISSUE THREAD

**Issue by mfranzon** [Lightning-AI/lit-llama#198]
Do you think that can be useful having a custom langchain wrapper for lit-llama? 
Using the CustomLLM class, proposed in the langchain docs: https://python.langchain.com/en/latest/modules/models/llms/examples/custom_llm.html

Thank you :)

**Comment by lantiga** [comment]
Hey @mfranzon, that would be cool! Is it something you'd be interested in contributing?

**Comment by themantalope** [comment]
Any progress on this? I would also like to use  it.

**Comment by mfranzon** [comment]
@themantalope Yep, I am trying to do at my best but in this period I have to many commitments, I hope to push a PR as soon as possible.
